src/Security/InheritedPermissions.php

```diff
@@ -631,7 +636,26 @@ protected function getPermissionField($type)
      * @param string $type
      * @return string
      */
-    protected function getJoinTable($type)
+    protected function getJoinTable($type, $permissionType = InheritedPermissions::ONLY_THESE_USERS)
```

Comment:
We can't add extra parameters (even if they have default values) in minor releases, since this means anyone crazy enough to subclass this thing and override this method would have to change the method signature in their class. It's technically a breaking change.